src/Broker/Broker.h

```diff
@@ -0,0 +1,22 @@
+#ifndef BROKER_H
+#define BROKER_H
+
+#include <system_spec.h>
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include <Packetizer.h>
```

Comment:
Maybe just include `SpacePacket.h` as we only use two defines here?

Reply:
Yes, this should be enaugh. `Packetizer.h` is will be included in `Broker.c`.

Reply:
done